### ISSUE TITLE
Antlr 4 grammar for Peoplecode 

### ISSUE THREAD

**Issue by tslater2006** [OpenPplSoft/OPS#1]
The grammar seems to not be able to handle having output parameters:

```
method Login( &clientId As string, &exp As datetime out, &locale As string out) Returns string;
```

Also it doesn't seem to accept strings with escaped quotes in it

```
local string &myString = "hello""world""";
```

the above is valid peoplecode and yields a string that contains (Hello "World").

**Comment by tslater2006** [comment]
I've sent a pull request to fix the first one, simply added "out" as an optional piece of the formalParamList rule.

**Comment by mquinn** [comment]
Resolved with the merge of OPS#2 into master; thank you for the contributions.
